src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -263,16 +257,136 @@ public static State createState(
 			"        		\"tx_errors\": 0,\n" +
 			"        		\"tx_packets\": 27\n" +
 			"      		},\n" +
-			"			\"iface\": \"wlan0\",\n" +
+			"			\"iface\": \"wlan%d\",\n" +
 			"			\"mode\": \"ap\",\n" +
 			"			\"phy\": \"platform/soc/c000000.wifi\",\n" +
 			"           \"radio\": {\n" +
-			"				\"$ref\": \"#/radios/0\"\n" +
+			"				\"$ref\": \"#/radios/%d\"\n" +
 			"			},\n" +
-			"			\"ssid\": \"OpenWifi_dddd\"\n" +
-			"		}\n" +
-			"	 ]\n" +
+			"			\"ssid\": \"OpenWifi_dddd_%d\"\n" +
+			"		 }\n" +
+			"	   ]\n" +
+			"    },\n",
+			index,
+			index,
+			index,
+			index,
+			index
+		);
+		// @formatter:on
+	}
+
+	/**
+	 * Create a device state object with one radio.
+	 * 
+	 * @param channel channel number
+	 * @param channelWidth channel width in MHz
+	 * @param bssid bssid
+	 * @return the state of an AP with one radio
+	 */
+	public static State createState(int channel, int channelWidth, String bssid) {
+		final int numRadios = 1;
+		String ifaceString = createStateInterfaceString(0);
+		StringBuffer stateString = new StringBuffer();
```

Comment:
nit: this is explicitly not a string, so don't name it like this - usually `StringBuffer` var is named `sb`